legion/jupyterlab-plugin/jupyter_legion/handlers/cloud.py

```diff
@@ -0,0 +1,252 @@
+#
+#    Copyright 2019 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
Check work dir training parameter. It is not propagated to MD resource

Reply:
Done